File: Volume 10 (1000 - 1099)/1015/1015.c
#include <stdio.h>
#include <math.h>

int main() {
    double x1, x2, y1, y2;
    scanf("%lf%lf", &x1, &y1);
    scanf("%lf%lf", &x2, &y2);
    printf("%.4lf\n", sqrt((x2 -x1)*(x2 -x1)+(y2 -y1)*(y2 -y1)));
    return 0;
}

File: Volume 10 (1000 - 1099)/1015/1015.py
#!/usr/bin/env python3

import math

p1 = x = [float(i) for i in (input().split(' '))]
p2 = x = [float(i) for i in (input().split(' '))]
distance = math.sqrt((p2[0]*p2[0]-2*p1[0]*p2[0]+p1[0]*p1[0]) + (p2[1]*p2[1]-2*p1[1]*p2[1]+p1[1]*p1[1]))
print("{:.4f}".format(distance))